app/lib/screens/identity_verification_screen.dart

```diff
@@ -619,6 +626,26 @@ class IdentityVerificationScreenState
 
   @override
   Widget build(BuildContext context) {
+    if (isLoading) {
+      return Scaffold(
```

Comment:
Why are we using a scaffold here? please use the LayoutDrawer and change the content

Reply:
Please add an error widget too